cmd/pg-schema-diff/plan_cmd.go

```diff
@@ -59,8 +86,13 @@ func buildPlanCmd() *cobra.Command {
 			fmt.Println("Schema matches expected. No plan generated")
 			return nil
 		}
-		fmt.Printf("\n%s\n", header("Generated plan"))
-		fmt.Println(planToPrettyS(plan))
+
+		if planFlags.outputFormat == "" || planFlags.outputFormat == outputFormatPretty {
```

Comment:
Let's not support an empty string for output format (once the param is default and required). Let's also error out if the enum value is not known, i.e., if neither if condition matches, then return an error.

Alternatively (up to you!), you could add to the output format type an additional property called `print`:
```
{
    name string
    print func(plan)
}
....
...
planFlags.outputFormat.print(plan)
```

Then we wouldn't need any sort of conditional switchhere.

Reply:
I reworked this as suggested - now each output format has a function to convert the plan to an output string.

Thanks for the suggestion, it's definitely much cleaner.